molgenis-data-mapper/src/main/java/org/molgenis/data/meta/AttributeMappingMetaData.java

```diff
@@ -6,7 +6,7 @@
 
 public class AttributeMappingMetaData extends DefaultEntityMetaData
 {
-	public static final String ENTITY_NAME = "attributes";
+	public static final String ENTITY_NAME = "AttributeMapping";
 	public static final String IDENTIFIER = "identifier";
 	public static final String SOURCEATTRIBUTEMETADATA = "sourceAttribureMetaData";
```

Comment:
I fancy keeping the name exactly like the java classname of the corresponding Object